app/plugin/TreeColumnStyleSwitcher.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * A plugin for Ext.grid.column.Column that extends the internal cellTpl to
+ * add a radio group to switch the style for a layer
+ * (CpsiMapview.view.layer.StyleSwitcherRadioGroup).
```

Comment:
```suggestion
 * (CpsiMapview.plugin.TreeColumnStyleSwitcher).
```

Reply:
This is meant as reference for class offering the RadioGroup mentioned in the text.